src/DynamoCore/Models/TypeLoadData.cs

```diff
@@ -127,5 +127,7 @@ public string Category
         ///     The description of this type.
         /// </summary>
         public readonly string Description;
+
+        public bool IsPackageMember;
```

Comment:
Since this class is all documented, let's continue on that good practice by adding this:

```csharp
/// <summary>
///     Indicates if the type is loaded off a package.
/// </summary>
```